frontend/src/components/creator-modal/creator-edit/creator-edit.tsx

```diff
@@ -162,7 +196,7 @@ const CreatorEdit = (): ReactElement => {
             <input
```

Comment:
The add answer choice button still needs to be given a tab index

Reply:
Why are you guys messing with the tab index? the default usually works very well

Reply:
some of the elements are divs